src/core/Track.cpp

```diff
@@ -637,4 +637,20 @@ BoolModel *Track::getMutedModel()
 	return &m_mutedModel;
 }
 
+void Track::setName( const QString & newName )
+	{
+		if (m_name != newName)
+		{
+			m_name = newName;
+
+			lmms::Song * song = Engine::getSong();
+			if (song)
+			{
+				song->setModified();
+			}
+			
+			emit nameChanged();
+		}
+	}
```

Comment:
```suggestion
void Track::setName(const QString& newName)
{
	if (m_name != newName)
	{
		m_name = newName;

		if (auto song = Engine::getSong())
		{
			song->setModified();
		}
		
		emit nameChanged();
	}
}
```
Fixed the indentation and cleaned up the style

Reply:
Fixed with commit 18114256.